openlibrary/catalog/marc/parse.py

```diff
@@ -380,72 +389,57 @@ def read_author_person(f):
     ]
     for subfield, field_name in subfields:
         if subfield in contents:
-            author[field_name] = remove_trailing_dot(
-                ' '.join([x.strip(' /,;:') for x in contents[subfield]])
-            )
-    if 'q' in contents:
-        author['fuller_name'] = ' '.join(contents['q'])
```

Comment:
Is this information ($q fuller form of name) being intentionally dropped?

Reply:
@tfmorris  Sorry, my dropping $q was a little bit sub-conscious. It will conflict with #7409 

I'm still not convinced the $q fuller name is a complete feature though. 

The $q subfield contains an unstructured name fragment which is associated with a particular name field on a particular record. We will be breaking that link and adding it as a stand-alone field on an author record. The author record may have it's main name changed, so it won't be clear which of the multiple alt names or main name the fuller (partial) form relates to.

$q seems to me to be a property of a specific name string, not of a person, nor even a particular record. With most other subfields, it's clearer how they relate directly to the subject of the record.  

I feel like I'm being very pedantic, but it seems less well designed than other subfields, and it shows in the variability of how the subfield is used in practice. It seems more ad-hoc than usual. Just the fact that it is named 'Fuller form of name', but its content is always a partial name which cannot stand alone seems poor design.

OL's import is going to lose the association and context of the original name string, which could further get confused by author merges.

I suppose I _am_ saying I'd rather see it removed than in. The `$0` and `$1` authority control subfields seem a much more valuable way forward, and clearly and directly relate to the person / subject of the record in a way that should be usable by automated processing tools - unlike `$q` .  (see #7724 , raised from you other comments on this PR)